src/pages/sighting/CommitBanner.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { get, flatten, every } from 'lodash-es';
+
+import useCommitAssetGroupSighting from '../../models/assetGroupSighting/useCommitAssetGroupSighting';
+import Button from '../../components/Button';
+import Alert from '../../components/Alert';
+
+const alertProps = {
+  severity: 'info',
+  style: { marginBottom: 24 },
+};
+
+export default function SightingCore({
```

Comment:
I know that it doesn't end up mattering, but I'd advocate re-naming this.

Reply:
oh totally